salt/metalk8s/orchestrate/upgrade/init.sls

```diff
@@ -53,9 +62,16 @@ Skip node {{ node }}, already in {{ node_version }} newer than {{ dest_version }
 
   {%- else %}
 
+Install apiserver-proxy on {{ node }}:
+  salt.state:
+    - tgt: {{ node }}
+    - sls:
+      - metalk8s.kubernetes.apiserver-proxy
+    - saltenv: {{ saltenv }}
+
 Wait for API server to be available on {{ node }}:
```

Comment:
This one should require `salt: Install apiserver-proxy on {{ node }}` we don't want to run this one if proxy do not work otherwise we may wait for a timeout for ... nothing

Reply:
Agree,  done.